packages/kbn-es/src/utils/docker.ts

```diff
@@ -621,7 +629,25 @@ export async function runServerlessCluster(log: ToolingLog, options: ServerlessO
 
   if (options.waitForReady) {
     log.info('Waiting until ES is ready to serve requests...');
-    await waitUntilClusterReady();
+
+    const esNodeUrl = `${options.ssl ? 'https' : 'http'}://${portCmd[1].substring(
+      0,
+      portCmd[1].lastIndexOf(':')
+    )}`;
+
+    await waitUntilClusterReady({
+      node: esNodeUrl,
+      ...(options.ssl
+        ? {
+            auth: { bearer: kibanaDevServiceAccount.token },
+            tls: {
+              ca: CA_CERT_PATH,
+              // Required for self signed cert
```

Comment:
Just a small comment: strange we still get self signed cert errors after setting up the CA . Have you checked if we need `rejectUnauthorized: false` after setting the CA?

Reply:
I did check, it is required. I think it is due to part of the certificate chain being self signed.